diff -urN ../PDa/src/makefile PDa/src/makefile
--- ../PDa/src/makefile	2007-02-16 11:54:32.000000000 +0100
+++ PDa/src/makefile	2008-10-22 15:37:25.000000000 +0200
@@ -19,10 +19,10 @@
 pd-gui_LDFLAGS = -L$(X11LIB)
 pd-gui_DEFINES = $(DEFINES)
 
-pd_LIBS = -lm -lpthread -ldl -lesd
+pd_LIBS = -lm -lpthread -ldl
 pd_LDFLAGS = $(LDFLAGS) -Wl,-export-dynamic
 pd_DEFINES = $(DEFINES) -DINSTALL_PREFIX=\"$(PREFIX)\" \
-             -DFIXEDPOINT -DUSEAPI_OSS -DUSEAPI_ESD -DDL_OPEN
+             -DFIXEDPOINT -DUSEAPI_OSS -DDL_OPEN
 
 extra_DEFINES = $(DEFINES) -DFIXEDPOINT
 extra_INCLUDES = -I../src
@@ -106,8 +106,8 @@
 
 static: 
 	make pd_SRC="$(pd_SRC) $(extra_SRC)" DEFINES="-DPD -DUNIX -DSTATIC" \
-	     pd $(GUINAME) pd-watchdog $(BIN_DIR)/pdsend \
-	     $(BIN_DIR)/pdreceive  $(BIN_DIR)/pd.tk
+	     pd pd-watchdog $(BIN_DIR)/pdsend \
+	     $(BIN_DIR)/pdreceive
 
 extra: $(extra_EXT)
 
@@ -119,6 +119,9 @@
 ipod:
 	make CC=arm-elf-gcc static
 
+zaurus:
+	make CFLAGS="-march=armv4 -mtune=strongarm -O3 -fomit-frame-pointer" static
+
 blackfin:
 	make CC=bfin-uclinux-gcc static
 
diff -urN ../PDa/src/s_loader.c PDa/src/s_loader.c
--- ../PDa/src/s_loader.c	2007-02-14 10:36:49.000000000 +0100
+++ PDa/src/s_loader.c	2008-10-22 15:31:53.000000000 +0200
@@ -48,7 +48,7 @@
 
 #ifdef STATIC
 int sys_load_lib(char *dirname, char *classname)
-{ return 1;}
+{ return 0;}
 #else
 int sys_load_lib(char *dirname, char *classname)
 {
